osu.Game/Screens/Play/HUD/GameplayAccuracyCounter.cs

```diff
@@ -1,20 +1,54 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
-
+using System.ComponentModel;
 using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
 using osu.Game.Graphics.UserInterface;
 using osu.Game.Rulesets.Scoring;
 
 namespace osu.Game.Screens.Play.HUD
 {
     public abstract partial class GameplayAccuracyCounter : PercentageCounter
     {
+        [SettingSource("Accuracy Display Mode", "Which Accuracy will display")]
+        public Bindable<AccuracyType> AccuracyDisplay { get; } = new Bindable<AccuracyType>();
+
         [BackgroundDependencyLoader]
         private void load(ScoreProcessor scoreProcessor)
         {
-            Current.BindTo(scoreProcessor.Accuracy);
+            AccuracyDisplay.BindValueChanged(mod =>
+            {
+                Current.UnbindBindings();
+
+                switch (mod.NewValue)
+                {
+                    case AccuracyType.Rolling:
+                        Current.BindTo(scoreProcessor.Accuracy);
+                        break;
+
+                    case AccuracyType.Increase:
+                        Current.BindTo(scoreProcessor.IncreaseAccuracy);
+                        break;
+
+                    case AccuracyType.Decrease:
+                        Current.BindTo(scoreProcessor.DecreaseAccuracy);
+                        break;
+                }
+            }, true);
+        }
+
+        public enum AccuracyType
+        {
+            [Description("Rolling")]
```

Comment:
dk how i feel about this, id have no idea what rolling does without seeing the code, " classic" or "base on previous hits" something of that sort would feel better IMO but YMMV

Reply:
I would say "Accumulative" as a good alternative, meaning the accumulation of judged hits to decide the accuracy.

Reply:
don't see how "accumulative" is better than "rolling". if both of those are too difficult to understand then i'd advocate for "classic" or "standard"